src/main/scala/AtomRenderer.scala

```diff
@@ -29,7 +29,7 @@ trait AtomRenderer {
   type JS = Option[String]
 
   def getHTML[A](atom: Atom, data: A, conf: Conf)(implicit reader: Rendering[A]): HTML =
-    reader.html(atom, data).toString
+    reader.html(atom, data)(conf.asInstanceOf[reader.Conf]).toString
```

Comment:
I don't know another way of reconciling the two types. I tried with an implicit evidence that `Conf <:< reader.Conf` but the compiler cannot prove it.

Reply:
as discussed, this can probably be fixed by refactoring to remove the need for an implicit param